File: PYTHON/petshop.py
class petshop:
    __ID = ""
    __namaproduk = ""
    __kategoriproduk = ""
    __hargaproduk = 0

    def __init__(self): #constructor
        self.ID = ""
        self.namaproduk = ""
        self.kategoriproduk = ""
        self.hargaproduk = 0

    #setter dan getter
    def setID(self, id):
        self.ID = id
    def setNamaproduk(self, nama):
        self.namaproduk = nama
    def setKategoriproduk(self, kategori):
        self.kategoriproduk = kategori
    def setHargaproduk(self, harga):
        self.hargaproduk = harga

    def getID(self):
        return self.ID
    def getNamaproduk(self):
        return self.namaproduk
    def getKategoriproduk(self):
        return self.kategoriproduk
    def getHargaproduk(self):
        return self.hargaproduk

File: JAVA/petshop.java
class petshop
{
    private String ID;
    private String namaproduk;
    private String kategoriproduk;
    private int hargaproduk;

    petshop()
    {
        this.ID = "";
        this.namaproduk = "";
        this.kategoriproduk = "";
        this.hargaproduk = 0;
    };

    petshop(String id, String nama, String kategori, int harga) //constructor
    {
        this.ID = id;
        this.namaproduk = nama;
        this.kategoriproduk = kategori;
        this.hargaproduk = harga;
    }
    //setter dan getter
    void setID(String id)
    {
        this.ID = id;
    }
    void setNamaProduk(String nama)
    {
        this.namaproduk = nama;
    }
    void setKategoriProduk(String kategori)
    {
        this.kategoriproduk = kategori;
    }
    void setHargaProduk(int harga)
    {
        this.hargaproduk = harga;
    }

    String getID()
    {
        return this.ID;
    }
    String getNamaProduk()
    {
        return this.namaproduk;
    }
    String getKategoriProduk()
    {
        return this.kategoriproduk;
    }
    int getHargaProduk()
    {
        return this.hargaproduk;
    }
};